src/modules/Builder/features/ActivityItems/ActivityItems.tsx

```diff
@@ -35,31 +36,23 @@ export const ActivityItems = () => {
   });
 
   const { appletId, activityId, itemId } = useParams();
-  const [activeItemIndex, setActiveItemIndex] = useState(() =>
-    itemId ? items?.findIndex((item) => getEntityKey(item) === itemId) : -1,
-  );
-  const [activeItem, setActiveItem] = useState<ItemFormValues | undefined>(undefined);
+  const activeItemIndex = itemId ? items?.findIndex((item) => getEntityKey(item) === itemId) : -1;
```

Comment:
as I see from this ternary, activeItemIndex can be also undefined, not only -1

Reply:
fixed